### ISSUE TITLE
Read exif data synchronously

### ISSUE THREAD

**Issue by Hesesses** [jseidelin/exif-js#39]
Is is somehow possible to read exif data synchronously?

```
EXIF.getData(origImage, function() {
    Orientation = EXIF.getTag(this, "Orientation");
});

switch (Orientation)
.... 
```

**Comment by systembugtj** [comment]
you can use promise to synchronize it.

**Comment by diegocr** [comment]
> Is is somehow possible to read exif data synchronously?

Yes, deal with the `<img/>`s data yourself and then call `EXIF.readFromBinaryFile()`

**Comment by Stephenradams** [comment]
I found using EXIF.getAllTags() returned everything I needed so I could add it to an object.

**Comment by Masterdanielsan** [comment]
can anyone explain to me how they accomplished this?  I'm having a hell of a time reading in EXIF for hundreds of files when it starts asynchronously handling everything.
